builder/cloudstack/step_create_instance.go

```diff
@@ -47,7 +47,9 @@ func (s *stepCreateInstance) Run(_ context.Context, state multistep.StateBag) mu
 	p.SetDisplayname("Created by Packer")
 
 	if keypair, ok := state.GetOk("keypair"); ok {
-		p.SetKeypair(keypair.(string))
+		kp := keypair.(string)
+		ui.Message(fmt.Sprintf("Found keypair: %s", kp))
```

Comment:
Maybe `Using keypair: %s` describes this a bit better (as the keypair is not found but specified in the config)?